deep-learning/README.md

```diff
@@ -1,6 +1,7 @@
-# Deep Learning Resources
+# Machine Learning and Deep Learning
 
```

Comment:
Are you sure about the Machine Learning Part ?
The majority of repo focuses on DL aspects only so better to jeep the name as Deep Leaning onlu